hera_cal/tests/test_lstbin.py

```diff
@@ -244,6 +273,16 @@ def test_lst_bin_files(self):
         os.remove(output_lst_file)
         os.remove(output_std_file)
 
+        # test weight_by_nsamples
+        lstbin.lst_bin_files(self.data_files, ntimes_per_file=250, outdir="./", overwrite=True,
+                             verbose=False, rephase=True, weight_by_nsamples=False, file_ext=file_ext)
+        output_lst_file = "./zen.ee.LST.0.20124.uvh5"
+        output_std_file = "./zen.ee.STD.0.20124.uvh5"
+        assert os.path.exists(output_lst_file)
+        assert os.path.exists(output_std_file)
+        os.remove(output_lst_file)
+        os.remove(output_std_file)
+
```

Comment:
Please also extract this into its own test

Reply:
Actually so for here I did not do anything I just tested there is no error running this new option so it kind of falls into the "# basic execution" catalogue, similar to all the tests above like testing the `rephase` option. Does this still need to be its separate test?